### PR TITLE
Increase size of prometheus disk for 2i2c-aws-us

### PR DESCRIPTION
**Pull request by yuvipanda**
Was full :(

### CODE REVIEW

**Comment by yuvipanda** [comment]
Already manually deployed

File: config/clusters/2i2c-aws-us/support.values.yaml [hunk 1]
```diff
@@ -27,9 +27,7 @@ prometheus:
           hosts:
             - prometheus.aws.2i2c.cloud
     persistentVolume:
-      # Found to be out of space with at 100Gi disk at 2023-07-31, when it was
-      # increased to 250Gi.
-      size: 250Gi
```
Comment (consideRatio) [inline config/clusters/2i2c-aws-us/support.values.yaml:32]: The growth rate seems like ~100GB/month, so the increase of 250GB gives us ~2.5 month of time then I figure.

**Review by consideRatio — APPROVED** [review]

**Comment by GeorgianaElena** [comment]
@consideRatio, let's also merge this to make sure it doesn't get overwritten?

**Comment by consideRatio** [comment]
Good call @GeorgianaElena!

File: config/clusters/2i2c-aws-us/support.values.yaml (hunk 1 above)
Comment (yuvipanda) [reply config/clusters/2i2c-aws-us/support.values.yaml:32]: @consideRatio I think that was mostly due to the small blocksize we had running, and I think this cluster never actually came back from that. If you look at the disk free space graph now:

image

Usage actually *shrunk* by about 100GB after prometheus came back and compacted all those blocks!

**Pull request merged**